translations/fr.po

```diff
@@ -93,258 +89,166 @@ msgstr ""
 msgstr "Sauvant les lemmes personalisés"
 
-#: config.py:243
-msgid "Catalan"
-msgstr "Catalan"
-
-#: config.py:243
-msgid "Danish"
-msgstr "Danois"
-
-#: config.py:243
-msgid "German"
-msgstr "Allemand"
-
-#: config.py:243
-msgid "Greek"
-msgstr "Grec"
-
-#: config.py:243
-msgid "English"
-msgstr "Anglais"
-
-#: config.py:243
-msgid "Finnish"
-msgstr "Finlandais"
-
-#: config.py:243
-msgid "French"
-msgstr "Français"
-
-#: config.py:243
-msgid "Serbo-Croatian"
-msgstr "Serbo-Croate"
-
-#: config.py:243
-msgid "Italian"
-msgstr "Italien"
-
-#: config.py:243
-msgid "Japanese"
-msgstr "Japonais"
-
-#: config.py:243
-msgid "Korean"
-msgstr "Coréen"
-
-#: config.py:243
-msgid "Lithuanian"
-msgstr "Lituanien"
-
-#: config.py:243
-msgid "Macedonian"
-msgstr "Macédonien"
-
-#: config.py:243
-msgid "Dutch"
-msgstr "Néerlandais"
-
-#: config.py:243
-msgid "Norwegian Bokmål"
-msgstr "Norvégien Bokmål"
-
-#: config.py:243
-msgid "Polish"
-msgstr "Polonais"
-
-#: config.py:243
-msgid "Portuguese"
-msgstr "Portugais"
-
-#: config.py:243
-msgid "Romanian"
-msgstr "Roumain"
-
-#: config.py:243
-msgid "Russian"
-msgstr "Russe"
-
-#: config.py:243
-msgid "Spanish"
-msgstr "Espagnol"
-
-#: config.py:243
-msgid "Swedish"
-msgstr "Suédois"
-
-#: config.py:243
-msgid "Ukrainian"
-msgstr "Ukrainien"
-
-#: config.py:243
-msgid "Chinese"
-msgstr "Chinois"
-
-#: config.py:246
-msgid "Select Wiktionary source language"
-msgstr "Sélectionnez la langue source du Wiktionary"
-
-#: config.py:247
-msgid "Language"
-msgstr "Langue"
-
-#: config.py:266
+#: config.py:262
 msgid "Downloading Wiktionary"
 msgstr "Téléchargeant le Wiktionary"
 
-#: config.py:339
+#: config.py:340
 msgid "Choose format manually"
 msgstr "Choisir le format manuellement"
 
-#: config.py:346
+#: config.py:347
 msgid "Create files for all available formats"
 msgstr "Créer les fichiers pour tous les formats disponibles"
 
-#: config.py:378
+#: config.py:379
 msgid "Choose book format"
 msgstr "Choisir le format du livre"
 
-#: config.py:384
+#: config.py:385
 msgid ""
 "This book has multiple supported formats. Choose the format you want to use."
 msgstr ""
 "Ce livre a plusieurs formats pris en charge. Choisissez le format que vous "
 "voulez utiliser."
 
-#: config.py:390
+#: config.py:391
 msgid "Always ask when more than one format is available"
 msgstr "Toujours demander quand il y a plus d'un format"
 
-#: custom_lemmas.py:51
-msgid "Customize"
-msgstr "Personaliser"
+#: config.py:414
+msgid "Choose Wiktionary language"
+msgstr "Sélectionnez la langue du Wiktionary"
+
+#: config.py:435 config.py:436
+msgid "Simplified Chinese"
+msgstr "Chinois simplifié"
 
-#: custom_lemmas.py:54
-msgid "Wiktionary"
-msgstr "Wiktionary"
+#: config.py:441
+msgid "Lemma language"
+msgstr "Langue des lemmes"
 
-#: custom_lemmas.py:57
-msgid "Kindle Word Wise"
-msgstr "Word Wise Kindle"
+#: config.py:447
+msgid "Gloss language"
+msgstr "Langue du Gloss (Global Language Online Support System)"
```

Comment:
"gloss" is not an acronym, it's just the short definition of a word(lemma).

Reply:
"Langue du gloss" OK then?

Reply:
Sure.

Reply:
Correct. Glose is a better translation. I learned a new word today :-)